org/github/pr-for-subtree.ts

```diff
@@ -0,0 +1,46 @@
+import {warn, danger} from "danger";
+
+/*
+    This rule scans a PR for changes in a specific subtree and adds
+    instructions for merging the changes back into the source repo.
+*/
+export default async () => {
+    
+    // The subtree path we monitor.
+    const subtreePath = "libs/login/";
+    
+    // Look for subtree changes in the PR.
+    console.log(`Scanning PR for changes in ${subtreePath}.`);
+    const containsSubtreeChanges = danger.git.modified_files.some(f => f.includes(subtreePath)) ||
+                                   danger.git.created_files.some(f => f.includes(subtreePath)) ||
+                                   danger.git.deleted_files.some(f => f.includes(subtreePath));
+
+    // If we found changes in the subtree folder, add instructions to the PR.
+    if (containsSubtreeChanges) {
+        console.log(`PR contains changes in ${subtreePath} subtree.  Adding merge instructions as a warning.`);
+        
+        // Where the subtree changes need to go.
+        const destRepo = "WordPress-Login-Flow-Android";
+        
+        // Repo owner.
+        const org = "Automattic";
```

Comment:
The `WordPress-Login-Flow-Android` repo is in the `wordpress-mobile` org, so we probably want this:

```suggestion
        const org = "wordpress-mobile";
```

Reply:
Whoops!  Good catch!  Fixed.